### PR TITLE
Provided command line argument for full ODE URL

### PR DESCRIPTION
**Pull request by hmusavi**
Added ode-upload-url command line argument so the test harness no longer relies on DOCKER_HOST_IP environment variable to reach the ODE. Renamed `full-test.sh` to `full-test-sample.sh`. The users must create their own `full-test.sh` according to the provided sample script.

### CODE REVIEW

File: qa/test-harness/README.md [hunk 1]
```diff
@@ -45,9 +45,13 @@ deactivate
 
 ### Option 1:  (Recommended) Using the pre-configured test script
 
-You may perform a complete test of all of the ODE input file types by running the pre-configured full-test script.
+You may perform a complete test of all of the ODE input file types by running the pre-configured `full-test-sample` script.
+However, you may need to modify the `--ode-upload-url` to match your specific ODE deployment. It is also strongly recommended
+to use your own test files and add additional tests to ensure a better a fuller coverage. To do so, copy the provided
+`full-test-sample` into a new file name such as `fulle-test.sh` and make all necessary and recommended modification to the new file.
```
Comment (mvs5465) [inline qa/test-harness/README.md]: full**E**-test.sh 

:)

File: qa/test-harness/README.md [hunk 2]
```diff
@@ -45,9 +45,13 @@ deactivate
...
+You may then run your specific and customized tests by running the following command.
 
-Run `./full-test.sh`
+`source ./full-test.sh`
```
Comment (mvs5465) [inline qa/test-harness/README.md]: Not sure if you meant to put source or shell here, should probably be `/bin/sh ./full-test-sample.sh`

File: qa/test-harness/test-harness.py [hunk 3]
```diff
@@ -13,16 +13,13 @@
 
 KAFKA_CONSUMER_TIMEOUT = 10000
 KAFKA_PORT = '9092'
-ODE_PORT = '8080'
-ODE_REST_ENDPOINT = '/upload/bsmlog'
 
 DOCKER_HOST_IP=os.getenv('DOCKER_HOST_IP')
```
Comment (mvs5465) [inline qa/test-harness/test-harness.py]: DOCKER_HOST_IP is also used to set the Kafka subscription. Should that be switched over to `ode_upload_url` as well?

Comment (hmusavi) [reply qa/test-harness/test-harness.py]: I don't think so. I think to subscribe to Kafka, we still need the local IP of the host where kafka broker is running. For now anyway. In the future, we may need to revisit the use of the Host IP for accessing Kafka if Kafka broker is deployed separately.

File: qa/test-harness/README.md (hunk 2 above)
Comment (hmusavi) [reply qa/test-harness/README.md]: `source` will run the script in the current shell whereas `/bin/sh` will launch a new shell to run it in. It should work either way.

File: qa/test-harness/README.md (hunk 1 above)
Comment (hmusavi) [reply qa/test-harness/README.md]: oops. Fill fix.

**Pull request merged**